<launch>

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find m_detector_noted)/config/kitti/kitti1.yaml" />
	<arg name="pred_file" default="/mnt/e/dataset/rosbag_m_detector/kitti-20240314T095516Z-002/kitti/sequences/0000/predictions4/"/>	
	<!-- <arg name="pc_file" default=""/>	 -->
	<arg name="pc_topic" default="/velodyne_points"/>	

	<param name="dyn_obj/pc_topic" type="string" value="$(arg pc_topic)" />
	<param name="dyn_obj/pred_file" type="string" value="$(arg pred_file)" />

	<node pkg="m_detector_noted" type="display_prediction_noted" name="display_prediction_noted" output="screen" />

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find m_detector_noted)/rviz/display.rviz" />
	</group>
 
  	  

</launch>